Python/Product/PythonTools/PythonTools/Intellisense/CompletionControl.xaml

```diff
@@ -33,19 +33,12 @@
                         Background="{DynamicResource {x:Static wpf:Controls.TooltipBackgroundKey}}"
                         TextElement.Foreground="{DynamicResource {x:Static wpf:Controls.TooltipTextKey}}">
                     <StackPanel Orientation="Vertical">
-                        <TextBlock>
-                            The completion DB needs to be refreshed.
-                        </TextBlock>
+                        <TextBlock Text="{x:Static common:Strings.CompletionControl_NeedsRefresh}"/>
                         <StackPanel Orientation="Horizontal" Margin="0 3 0 0">
-                            <TextBlock Padding="0 0 3 0">
-                                Open
-                            </TextBlock>
-                            <Button Style="{StaticResource NavigationButton}" GotFocus="Button_GotFocus">
-                                Python Environments
-                            </Button>
-                            <TextBlock Padding="3 0 0 0">
-                                for more details.
-                            </TextBlock>
+                            <Button Margin="0 3 0 0"
+                                    Style="{StaticResource NavigationButton}"
+                                    GotFocus="Button_GotFocus"
+                                    Content="{x:Static common:Strings.CompletionControl_OpenEnvironmentsWindowForDetails}"/>
                         </StackPanel>
```

Comment:
Note: Small price to pay for localization, the whole string is now clickable instead of just Python Environments.